.github/workflows/acceptance-test.yaml

```diff
@@ -0,0 +1,87 @@
+# Run acceptance test every 12 hours
+name: Acceptance test
+
+on:
+  schedule:
+    - cron: '0 0,12 * * *'
+
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: true
+        default: 'info'
+        type: choice
+        options:
+        - info
+        - warning
+        - debug
+
+# Testing only needs permissions to read the repository contents.
+permissions:
+  contents: read
+
+jobs:
+  # Ensure project builds before running testing matrix
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
```

Comment:
Nit: why not simply do `actions/checkout@v4.1.0` instead? Same for the rest of the action version pinning.